components/frontend/src/App.js

```diff
@@ -265,11 +265,14 @@ class App extends Component {
               <Segment basic placeholder loading size="massive" />
               :
               this.state.report_uuid === "" ?
-                <Reports
-                  open_report={(e, r) => this.open_report(e, r)}
-                  reports_overview={this.state.reports_overview}
-                  {...props}
-                />
+                <>
+                  <Reports
+                    datamodel={this.state.datamodel}
+                    open_report={(e, r) => this.open_report(e, r)}
+                    reports_overview={this.state.reports_overview}
+                    {...props}
+                  />
+                </>
```

Comment:
Why is `<Reports>` embedded in a fragment?

Reply:
mh good question...